### PR TITLE
Harmonized line length style, spell-checked README

### PR DESCRIPTION
**Pull request by habi**
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

- [x] Improvement

- I've made the line length more consistent (by 'un-blocking' a paragraph)
- I've made minor changes to the help text of some options (and also changed them in the options.py file).
- As part of 24pullrequests.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,7 +183,7 @@ which means you can modify it, redistribute it or use it however you like.
                                      of SIZE.
     --playlist-reverse               Download playlist videos in reverse order
     --xattr-set-filesize             Set file xattribute ytdl.filesize with
-                                     expected filesize (experimental)
+                                     expected file size (experimental)
```
Comment (dstftw) [inline README.md]: Options are autogenerated.

Comment (habi) [inline README.md]: Autogenerated from where?
I was not able to find anything else than in the README and in options.py: https://github.com/rg3/youtube-dl/search?utf8=%E2%9C%93&q=--xattr-set-filesize&type=Code

Comment (habi) [inline README.md]: Thanks, that was not obvious.
Shall I change it back in the README, or is the [*actual* change in options.py](https://github.com/habi/youtube-dl/commit/8fbfdb3ae926c6162e06643406a7dc9080f3f454#diff-09f445e8685f0e866d7ff74c9481feec) anyways going to overwrite the README?

Comment (dstftw) [inline README.md]: It will overwrite on the next release. There is no point changing options in README manually.

**Comment by habi** [comment]
> It will overwrite on the next release. There is no point changing options in README manually.

Great!
I’ve learned something today :)

**Pull request closed without merging**